views/styles.html

```diff
@@ -340,5 +340,16 @@ <h2 class="_style-guide--title">Tabular Data</h2>
     {{table(tableData)}}
   </section>
 
+  <section class="_style-guide--group">
+    <h2 class="_style-guide--title">All Content Types</h2>
+    {% from "components/card.html" import card %}
+    <div class="card--group">
```

Comment:
Created the `layouts` stuff, but I'm not sure I'm calling it right in `styles.html`. I gave it a try and you can tell me if it's wrong.